starter-core/src/main/java/io/micronaut/starter/feature/json/SerializationBsonFeature.java

```diff
@@ -38,4 +48,31 @@ public String getTitle() {
     public String getModule() {
         return "bson";
     }
+
+    @Override
+    @NonNull
+    public List<Substitution> substitutions(@NonNull GeneratorContext generatorContext) {
```

Comment:
bson cant really substitute json, not sure if it really makes sense to exclude jackson here (though youd probably also be using serde json if youre using this feature)

Reply:
@sdelamo see my comment about that above. I have a PR to change the docs.

Reply:
I have a PR for that at https://github.com/micronaut-projects/micronaut-serialization/pull/195

Reply:
Ah yea, i forgot that the bson module can just use the bson library for serialization of json, still.